doc/nrf/protocols/matter/getting_started/low_power_configuration.rst

```diff
@@ -35,52 +94,35 @@ The Thread protocol defines two types of devices working in a low power mode: Sl
 Both types are variants of Minimal Thread Device (MTD) type.
 For more information about MTD and SEDs, see :ref:`Thread device types <thread_ot_device_types>`.
 
+The Matter ICD leverages Thread SED behavior, however these two terms are not necessarily related to each other and should not be confused.
```

Comment:
I think the previous version described the relation more accurately. The ICD and SED are different concepts describing behavior on a different abstraction level. Nevertheless in case of Thread, usage of ICD in Matter sort of implies enabling SED on a Thread network layer.

Reply:
Would this work
```suggestion
In a Thread network, :ref:`Matter ICD <ug_matter_device_low_power_icd>` behave like Thread SED, but the two terms are not interchangeable.
```